hack/istio/cypress/bookinfo-status.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+##############################################################################
+# bookinfo-status.sh
+#
+# Checks, if the Bookinfo application is installed. This script is used by the
+# Cypress Bookinfo before hook (frontend/cypress/integration/common/hooks.ts).
+# 
+##############################################################################
+set -e
+
+apps=("details-v1"
+      "kiali-traffic-generator"
+      "productpage-v1"
+      "ratings-v1"
+      "reviews-v1"
+      "reviews-v2"
+      "reviews-v3")
+
+input=$(kubectl get pods -n bookinfo -o=custom-columns=NAME:.metadata.name,Status:.status.phase --no-headers=true)
+
+for pod in ${!apps[@]}; do
+  count=$(echo "$input" | grep "${apps[$pod]}" | awk -F ' ' '{print $1}' | wc -l)
+  status=$(echo "$input" | grep "${apps[$pod]}" | awk -F ' ' '{print $2}')
+  if [ $count -ne 1 ] || [ $status != "Running" ]
```

Comment:
Pod could be also Pending I believe. We should not fail if the pod is still starting and wait for Running status.